src/pages/comparison/comparison.tsx

```diff
@@ -1,5 +1,268 @@
-import {ConstructionInfo} from "../../components"
+import { useState } from "react";
+import DeckGL from "@deck.gl/react";
+import {
+  MapController,
+  FlyToInterpolator,
+  COORDINATE_SYSTEM,
+  MapView,
+  WebMercatorViewport,
+} from "@deck.gl/core";
+import { StaticMap } from "react-map-gl";
+import { Tileset3D } from "@loaders.gl/tiles";
+import { Tile3DLayer } from "@deck.gl/geo-layers";
+import styled from "styled-components";
+import { getCurrentLayoutProperty, useAppLayout } from "../../utils/layout";
+import { I3SLoader } from "@loaders.gl/i3s";
+import { getElevationByCentralTile } from "../../utils";
+import { INITIAL_MAP_STYLE } from "../../constants/map-styles";
+
+interface layoutProps {
+  layout: string;
+}
+
+const INITIAL_VIEW_STATE = {
+  longitude: -120,
+  latitude: 34,
+  height: 600,
+  width: 800,
```

Comment:
`height` and `width` don't seem a part of viewState https://deck.gl/docs/api-reference/core/map-view#view-state

Reply:
removed them